### DISCUSSION TITLE
Cloze deletions

### DISCUSSION THREAD

**Discussion by mskfb** [st3v3nmw/obsidian-spaced-repetition#1055] [Q&A]
Hi - **zero** coding experience here, but using obsidian and really hopeful to integrate a spaced repetition system with my notes with this plugin.

This is the issue I am having:

I try to make a cloze flashcard using the default == == syntax, like this:
<img width="744" alt="Screenshot 2024-08-22 at 4 36 48 PM" src="https://github.com/user-attachments/assets/4b66226d-58d5-4512-a63a-f099efa6d262">

When I try to review this card, it then looks like this:
<img width="553" alt="Screenshot 2024-08-22 at 4 37 53 PM" src="https://github.com/user-attachments/assets/a27e4bc6-59e0-4f6a-8df6-9c75afed943d">

And then when I go back to the note the card was generated from (or after using regular inline flaschards with the :: syntax, which display well during review), the note itself displays like this:
<img width="765" alt="Screenshot 2024-08-22 at 4 38 32 PM" src="https://github.com/user-attachments/assets/f1352365-2c48-4212-a6bc-ede8f9210c40">

I suspect this has to do with my previous use of the "Outliner" plug-in, but I'm not sure how to undo this. I've disabled the plugin and the issue remains. When I view the note under "source view", nothing hidden seems to appear. 

Any ideas what the problem is and how it can be fixed?

Thanks

mskfb

**Comment by ronzulu** [comment]
Hi @mskfb 

> I suspect this has to do with my previous use of the "Outliner" plug-in, but I'm not sure how to undo this. I've disabled the plugin and the issue remains. When I view the note under "source view", nothing hidden seems to appear.

The following post may be helpful: https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/956#issuecomment-2119565705

If it is the same issue, then you will need to delete the tab character or space characters at the beginning of the line.
